docs/build_description.adoc

```diff
@@ -237,6 +237,28 @@ Only a single repository per product is usable. In case source
 or debug rpm's need to be added, they need to be part of the
 main repository.
 
+===== base_embed_packages
+
+To be used in conjunction with `base`.
+Controls whether packages should be copied in the `/install` directory
+inside the base iso images.
+
+Most of the time, you would want to include them, which is the
+default setting.
+
+Disabling this would result in a simply repacked base image, without
+any package copied there.
+
+Use either
+
+  base_embed_packages: include
+
+or
+
+  base_embed_packages: drop
```

Comment:
IMHO this should become a switch in build_options: instead, no need to deal with include/drop strings then.

But can't you achieve that by an empty package list as well?